exporters/zipkin/test/zipkin_exporter_test.cc

```diff
@@ -3,7 +3,6 @@
 
 #ifndef HAVE_CPP_STDLIB
 
-#  define _WINSOCKAPI_  // stops including winsock.h
```

Comment:
Not related to the PR:
This is not part of CMake test for Windows CI.

Reply:
I moved this `define` to spin lock header which includes `windows.h` as well. So this should be unnecessary. Let me know you prefer keep this define here.

Reply:
thanks, spin lock header is actually the right place to have the define.